pom.xml

```diff
@@ -206,7 +188,6 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps-global-lib</artifactId>
-            <version>2.12</version>
             <scope>test</scope>
             <exclusions>
                 <exclusion>
```

Comment:
Could I guess be deleted if you are pinning a `commons-lang3` version anyway (for some reason).